src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -183,6 +186,12 @@
iTDlmP6YVcwFzCzOEeY/FYjWw5lg3bCg2GZuJXYbdiN2FbcaewfZhB7FjOBxOC2eN88ZF4Vg4Ca4ItxN3EHca148bwn3Ek/AGeEd8MD4FL8CvwZfjG/Cn8P34Z/hxggrBlOBJiCJwCEsJmwn7CJ2Ea4QhwjhRlWhO9CbGETOJq4kVxCbiBeJD4lsSiWRE8iDNIfFJq0gVpMOkS6QB0ieyGtmKHECeR5aSN5EPkM+Q75HfUigUM4ofJYUioWyi1FPOUR5TPipRleyUmEocpZVKVUptSv1Kr5QJyqbKDOUFyvnK5cpHla8pv1QhqJipBKiwVFaoVKkcV7mjMqZKVXVQjVLNUd2o2qB6WXVYDadmphakxlErVNurdk5tkIqiGlMDqGzqWuo+6gXqEA1LM6cxaZm0EtohWi9tVF1N3Vk9QX2JepX6SXWZBkrDTIOpka2xWaNF47bG5xl6MxgzuDM2zGia0T/jg6aOpp8mV7NYs1nzluZnLbpWkFaW1latdq1H2mhtK+052ou1d2tf0H6pQ9Px0mHrFOu06NzXhXWtdGN0l+nu1e3RHdPT1wvRE+rt1Dun91JfQ99PP1O/TP+U/ogB1cDHgG9QZnDa4Dldnc6gZ9Mr6Ofpo4a6hqGGUsMaw17DcSNzo3ijNUbNRo+MicbuxunGZcZdxqMmBiaRJgUmjSb3TQmm7qY80x2m3aYfzMzNEs3Wm7WbDZtrmjPN880bzR9aUCx8LRZZ1FrctMRaultmWe6yvG4FW7lY8ayqrK5Zw9au1nzrXdZ9NhgbDxuBTa3NHVuyLcM2z7bRdsBOwy7Cbo1du92rmSYzU2Zundk985u9i322/T77Bw5qDmEOaxw6Hd44WjmyHascbzpRnIKdVjp1OL12tnbmOu92vutCdYl0We/S5fLV1c1V5NrkOuJm4pbqVu12x53mHu2+0f2SB8bD32OlxwmPT56unhLPFs+/vWy9srwavIZnmc/izto3a9DbyJvlXeMt86H7pPr84iPzNfRl+db6PvEz9uP47fd7xrBkZDIOMl752/uL/Fv9PwR4BiwPOBOICgwJLA7sDVILig+qDHocbBScEdwYPBriErIs5EwoJjQ8dGvoHaYek82sZ46GuYUtDzsfTg6PDa8MfxJhFSGK6IyEI8Mit0U+nG06WzC7PQpEMaO2RT2KNo9eFP3bHOyc6DlVc57GOMQUxHTHUmMXxjbEvo/zj9sc9yDeIl4a35WgnDAvoT7hQ2JgYmmiLGlm0vKkq8nayfzkjhRcSkLK/pSxuUFzt88dmucyr2je7fnm85fMv7xAe0H2gpMLlReyFh5NxaQmpjakfmFFsWpZY2nMtOq0UXYAewf7BcePU8YZ4XpzS7nP0r3TS9OHM7wztmWM8Hx55byX/AB+Jf91ZmjmnswPWVFZB7Lk2YnZzTn4nNSc4wI1QZbgfK5+7pLcPqG1sEgoW+S5aPuiUVG4aL8YEs8Xd0hoSDPUI7WQrpMO5PnkVeV9XJyw+OgS1SWCJT1LrZZuWPosPzj/12XoZexlXQWGBasLBpYzltesgFakrehaabyycOXQqpBVdauJq7NW/77Gfk3pmndrE9d2FuoVriocXBeyrrFIqUhUdGe91/o9P6F/4v/Uu8Fpw84N34o5xVdK7EvKS75sZG+88rPDzxU/yzelb+rd7Lp59xbsFsGW21t9t9aVqpbmlw5ui9zWVkYvKy57t33h9svlzuV7dhB3SHfIKiIqOnaa7Nyy80slr/JWlX9Vc7Vu9YbqD7s4u/p3++1u2qO3p2TP51/4v9ytCalpqzWrLd+L3Zu39+m+hH3dv7r/Wr9fe3/J/q8HBAdkdTF15+vd6usbdBs2N8KN0saRg/MOXj8UeKijybapplmjueQwOCw9/PxI6pHbLeEtXUfdjzYdMz1W3UptLW6D2pa2jbbz2mUdyR19x8OOd3V6dbb+ZvfbgROGJ6pOqp/cfIp4qvCU/HT+6bEzwjMvz2acHexa2PXgXNK5m+fnnO+9EH7h0sXgi+e6Gd2nL3lfOnHZ8/LxK+5X2q+6Xm3rcelp/d3l99Ze1962a27XOq57XO/sm9V3qt+3/+yNwBsXbzJvXr01+1bf7fjbd+/MuyO7y7k7fC/73uv7effHH6x6iHlY/EjlUflj3ce1f1j+0SxzlZ0cCBzoeRL75MEge/DFn+I/vwwVPqU8LX9m8Kx+2HH4xEjwyPXnc58PvRC+GH9Z9JfqX9WvLF4d+9vv757RpNGh16LX8jcb32q9PfDO+V3XWPTY4/c578c/FH/U+lj3yf1T9+fEz8/GF3/Bfan4avm181v4t4fyHLlcyBKxvrcCKMTh9HQA3hwAgJIMABXpiYlzJ3vo7wZN9v3fCfwvnuyzv5srAE3IMNEKMVYB0OKHtLCIKyEehXicH4CdnBT+bxOnOzlOaik1AoAzlMvf5AJAQPxLiFw+Hi2Xf61Gir0JwKnhyd59wrDIH00T1SFTh9f/TVID/sv+BYmREGAWHocYAAAABHNCSVQICAgIfAhkiAAAAF96VFh0UmF3IHByb2ZpbGUgdHlwZSBBUFAxAAAImeNKT81LLcpMVigoyk/LzEnlUgADYxMuE0sTS6NEAwMDCwMIMDQwMDYEkkZAtjlUKNEABZgamFmaGZsZmgMxiM8FAEi2FMk61EMyAAAA2klEQVRIiWM0s7T7z8DAwCAiIvzH1Nhoi6SEaHVGRsY1BhygvqF5/emz53zevHnLwsDAwMBgZmn3PyUt+wUuDbhASlr2CzNLu/8MXr6Bv0nVDANevoG/mUyNjbaQa4CpsdEWJkkJ0WpyDaBEL/UAI7rA/////+PVwMiIooeJ2i4aADBjxgwtSvQyPX/xupVcA56/eN3KdPrsOR9yDTh99pwP05s3b1nIzUxv3rxlYaQ0O1OeDoZ5efAfCnDJEywPCBnAwECFWBjk5QFdwoCJpuUBIxTgkh8m5QEA4ESeKl1VgwMAAAAASUVORK5CYII=');
```

Comment:
![image](https://user-images.githubusercontent.com/1047696/55644551-8b1c0d00-57d6-11e9-94b7-fdc04c0673a0.png)

can we use single toggle icons?